assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -129,6 +136,13 @@ function FontControls({ selectedElements, pushUpdate }) {
 
   const FontPickerDropdown = hasNewFontPicker ? FontPicker : DropDown;
 
+  usePresubmitHandler(
+    ({ fontSize: newFontSize }) => ({
+      fontSize: setMinMax(newFontSize, MIN_MAX.FONT_SIZE),
+    }),
+    []
+  );
```

Comment:
Why do it here and not in the `onChange` handler? What's the difference?

Reply:
Hey @barklund, `onChange` doesn't allow empty values while typing. I even tried this, but for instance, if your minimum is `0` and the field has an initial value of `200`...You can't empty the field or remove the value to type `120` given the reactive behavior of `onChange` event.

#### Algorithm flow using `onChange`

Context:

- The initial value of a given field is 200 and the field wants to enforce a minimum of 0.
- I want to add 120 for the field.

Failing Case 1:
1. Select field content and try to empty, it will add `0`.
2. Try to remove the `0` previously added to add the desired value `120`, it will add `0` again given the reactivity behavior of the onChange event, which will not allow adding `1`, then `2`, and finally `0` to reach the expected behavior.

Failing Case 2:
1. Select field content and then.
2. Delete each `200` character. First `0`, then `0`, and finally `2`
3. It will add automatically `0`, which will not allow adding `1`, then `2`, and finally `0` to reach the expected behavior.

On the other hand, `usePresubmitHandler ` uses the post-submit data manipulation pattern, which avoids the failing cases above. That's the main difference.

Reply:
Great, sounds good to me. And yes, we want people to be able to empty the contents, to type something else, so great that it's supported!